NEWS

```diff
@@ -11,9 +11,156 @@
+ * Added a reason when a resource's lane is unreachable to request query info string
+ * Added IB MLX5 as a separate UCX module with RPM
+ * Added initial GGA support
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
+ * Optimized XGVMI flow using user-mode registration cache
+ * Improved various logging messages
+#### CUDA
+ * Added multi-node NVLINK support
+ * Added CUDA Fabric memory support with detection and allocation
+ * Improved GDR latency estimations on AMD Milan systems
+ * Improved reliability by detecting runtime/build GDR library version match
+ * Added handling missing IPC capability when unpacking keys
+ * Added caching for CUDA IPC memory pool import operation
+ * Added GDR environment variables to optimize performance on Grace Hopper systems
+ * Improved peer accessible cache for CUDA IPC to be reentrant
+ * Improved CUDA IPC concurrency for a larger count of reachable peers
+#### UCS
+ * Improve non-temporal data transfer implementation
+ * Allowed configuration with wildcard prefix and name matching
+ * Defined a non-inline function to allow ASAN validation
+ * Reduced stack size in topology detection code
+ * Improved bitmaps configuration parsing with wider bitfield
+ * Added options to set topology distance between devices
+ * Optimized VFS unix socket watch by using user private folder
+ * Added general subnet matching support infrastructure
+ * Allowed user-based existing data copy when appending to array
+ * Improved time units description
+#### UCM
+ * Added memory hooks to track allocations by CUDA memory mapping functions
+#### Tools
+ * Improved performance by increasing window size for put_bw and add get_bw in ucx_perftest
+ * Added multi-send flag for receive operations in bandwidth benchmarks in ucx_perftest
+ * Improved ucx_perftest uni-directional test with added fence
+ * Improved ucx_perftest batch helper notes
+#### Documentation
+ * Added a section regarding adaptive routing on RoCE
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+ * Added AMD Turin support
+#### Build
+ * Improved compiler error reporting with added flag
+ * Improved coverity script to allow faster turnaround time
+ * Improved Intel Compiler detection and support
+#### GO
+ * Added multi-send flag and user memh support in request params
+#### Packaging
+ * Improved dpkg-buildpackage sample command by using explicit arguments
+### Bug
+#### UCP
+ * Fixed stack overflow in exported rkey unpack
+ * Removed extra remote-cpu overhead from protocol estimation for zcopy
+ * Fixed performance estimation for rndv pipeline protocols
+ * Fixed ATP sending by picking the correct lane
+ * Fixed missing reg_id on memh creation
+ * Fixed repeated invalidations by retaining existing access flags
+ * Fixed abort reason propagation for rendezvous RTR mtype
+ * Fixed transport availability for configuration list in ALLOW mode
```

Comment:
not clear?

Reply:
fixed with "Do not check transport availability if it is prevented by UCX_TLS environemnt variable"